reflex/components/chakra/media/icon.py

```diff
@@ -37,9 +37,9 @@ def create(cls, *children, **props):
             raise AttributeError(
                 f"Passing children to Icon component is not allowed: remove positional arguments {children} to fix"
             )
-        if "tag" not in props.keys():
+        if "tag" not in props:
             raise AttributeError("Missing 'tag' keyword-argument for Icon")
-        if type(props["tag"]) != str or props["tag"].lower() not in ICON_LIST:
+        if isinstance(props["tag"], str) or props["tag"].lower() not in ICON_LIST:
```

Comment:
This change fails on almost all CI.
I don't know if this is ruff doing or a manual change, but the equivalent of `type(props["tag"] != str)` should be `not isinstance(props["tag"], str)`

```suggestion
        if not isinstance(props["tag"], str) or props["tag"].lower() not in ICON_LIST:
```

Reply:
You are right, my bad...